README.md

```diff
@@ -51,6 +51,38 @@ to show the container's journal information on the host, using journalctl.
 journalctl -M CONTAINER_UUID
 ```
 
+To use this directly with runc, modify or add the following to config.json.
+
+    "hooks": {
+        "prestart": [
+            {
+                "path": "/usr/libexec/oci/hooks.d/oci-systemd-hook"
+            }
+        ]
+    },
+...
```

Comment:
I'm not sure what you're trying to do here with the dots and also on the three lines below.  Hit the view button and take a look at what it will look like if it was merged.  This section is pretty much unreadable.